src/js/components/TextInput.jsx

```diff
@@ -1,8 +1,13 @@
 import React, { useState } from "react";
-import tw from "twin.macro";
+import styled from "@emotion/styled";
```

Comment:
Are we moving towards styled components or tailwind?

Reply:
Since we using a new bundler, we have to update config. And to be consistent for now with what is already here in codebase I just moved these two or three lines to normal css.